recipe.py

```diff
@@ -23,17 +23,39 @@ def get_no_servings(self):
     def set_no_servings(self, no_servings):
         if int(no_servings):
             self.no_servings = no_servings
-            print("This is an integer")
+            #print("This is an integer")
         else:
             print("Please input an integer")
 
+    def get_calories_per_portion(self):
+        return self.calories_per_portion
+    
+    def set_calories_per_portion(self, calories_per_portion):
+        if int(calories_per_portion):
+            self.calories_per_portion = calories_per_portion
+        else:
+            print("Please input an integer")
+    
+    def get_ingredients(self):
+        return self.ingredients
+    
+    def set_ingredients(self,ingredients):
+        self.ingredients.append(ingredients)
```

Comment:
It could be argued that a duplication of ingredients should be avoided (and a check put in here) - but realistically I think its out of scope for this task.

Reply:
I will look at avoiding duplication, I'd like to add ingredients as a dictionary so the amount and ingredient can be stored and if then the duplication will be caught if there are duplicate keys

Reply:
looking at the ingredients class, I will leave this as a list and just avoid duplication